<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anApp1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>process</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_exe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>100</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OpenProcess</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pa1">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>771</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>recive message</string>
    </property>
    <widget class="QTextEdit" name="textEdit_reciv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>751</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>771</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>send message</string>
    </property>
    <widget class="QPushButton" name="pushButton_send">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SendMessage</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_send">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>751</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
